src/library_exceptions.js

```diff
@@ -329,6 +451,7 @@ var LibraryExceptions = {
 addCxaCatch = function(n) {
   LibraryManager.library['__cxa_find_matching_catch_' + n] = LibraryExceptions['__cxa_find_matching_catch'];
   LibraryManager.library['__cxa_find_matching_catch_' + n + '__sig'] = new Array(n + 2).join('i');
+  LibraryManager.library['__cxa_find_matching_catch_' + n + '__deps'] = LibraryExceptions['__cxa_find_matching_catch__deps'];
```

Comment:
this looks correct - I'm curious if it was necessary, though? (if so, I wonder why it wasn't necessary before)

Reply:
Yes, this was a bug. I found it when tried to add new dependency to `__cxa_find_matching_catch` and the resulted JS file did not contain new dependency function. This fixed it. Actually it worked by accident, it happened that all `__cxa_find_matching_catch` dependencies were satisfied by other previous functions.

Reply:
Makes sense, thanks for the info!